src/executor.ts

```diff
@@ -49,21 +49,21 @@ process.on('exit', () => {
   }
 });
 
-const spawnServer = (cwd, serverPath, debugOpt, logger) => {
-  server = spawn('node', [debugOpt, serverPath], { stdio: [0, 1, 2], cwd });
-  logger(`Spawning ${['node', debugOpt, serverPath].join(' ')}`);
+const spawnServer = (cwd, serverPath, options: { debugOpt: any; nodeDebugger: boolean }, logger) => {
```

Comment:
We should pass `args: string[]` here instead of `serverPath, options: { debugOpt: any; nodeDebugger: boolean }` and assemble proper `args` inside `runServer`

Reply:
@vlasenko I see your point as both falls under the same category. Let me make that change.